src/base/AsyncCallQueue.h

```diff
@@ -27,6 +27,10 @@ class AsyncCallQueue
     bool fire();
 
 private:
+    AsyncCallQueue() = default;
+    AsyncCallQueue(const AsyncCallQueue &) = delete;
+    AsyncCallQueue &operator=(const AsyncCallQueue &) = delete;
```

Comment:
We normally `delete` prohibited methods publicly, but I am OK with making an exception in this special case because we have to keep the default constructor private (for other reasons).

Reply:
>  if we can prohibit copying of AsyncCallLists 

We can - so I removed these two lines.